service_PlayAudio_name: "~PlayAudio"
service_SetVolume_name: "~SetVolume"
service_GetVolume_name: "~GetVolume"

default_volume: 50

mpd_host: "/run/mpd/socket"
mpd_port: 6600

update_hz: 10